docs/changelog.rst

```diff
@@ -8,6 +8,45 @@ This section lists features and improvements of note in each release.
 
 The full release history can be viewed `at the GitHub perses releases page <https://github.com/choderalab/perses/releases>`_.
 
+0.10.3 - Release
+----------------
+
+Bugfix release.
+
+
+Enhancements
+^^^^^^^^^^^^
+
+* Print perses version on startup by @mikemhenry in https://github.com/choderalab/perses/pull/1176
+* Support pymbar 3 & 4 by @mikemhenry in https://github.com/choderalab/perses/pull/1173
+* Store initial and final topologies for all phases -- small molecule pipeline by @ijpulidos in https://github.com/choderalab/perses/pull/1210
+
+Documentation
+^^^^^^^^^^^^^
+
+* Added MPI example with dipeptide mutation by @ijpulidos in https://github.com/choderalab/perses/pull/1228
+* Update comments in `RESTCapableHybridTopologyFactory` doc string by @zhang-ivy in https://github.com/choderalab/perses/pull/1189
```

Comment:
It is a minor thing but just to be consistent. In restructured Text the way to get inline monotyped/code blocks is with double ``
```suggestion
* Update comments in ``RESTCapableHybridTopologyFactory`` doc string by @zhang-ivy in https://github.com/choderalab/perses/pull/1189
```
Same applies to all the other instances.